<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWindow</class>
 <widget class="QDialog" name="DeviceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Karta urządzenia</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Opis</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Typ urządzenia</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="idEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Identyfikator urządzenia</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="locationEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="typeEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Stanowisko</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPlainTextEdit" name="descriptionEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Drukuj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="text">
            <string>Zamknij</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Atrybuty</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableView" name="attrTable"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="addAttrButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-add.png</normaloff>:/icons/icons/list-add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="deleteAttrButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-remove.png</normaloff>:/icons/icons/list-remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="editAttrButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-edit.png</normaloff>:/icons/icons/list-edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>133</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Podzespoły</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableView" name="componentTable"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="addComponentButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-add.png</normaloff>:/icons/icons/list-add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="deleteComponentButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-remove.png</normaloff>:/icons/icons/list-remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="editComponentButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-edit.png</normaloff>:/icons/icons/list-edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>136</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Konserwacja</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableView" name="conservationTable"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="addConservationButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-add.png</normaloff>:/icons/icons/list-add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="deleteConservationButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-remove.png</normaloff>:/icons/icons/list-remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="editConservationButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-edit.png</normaloff>:/icons/icons/list-edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>136</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Awarie</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="addAccidentButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-add.png</normaloff>:/icons/icons/list-add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="deleteAccidentButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-remove.png</normaloff>:/icons/icons/list-remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="editAccidentButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-edit.png</normaloff>:/icons/icons/list-edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>136</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Inne zdarzenia</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTableView" name="tableView_2"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-add.png</normaloff>:/icons/icons/list-add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-remove.png</normaloff>:/icons/icons/list-remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/list-edit.png</normaloff>:/icons/icons/list-edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>136</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
